Tests/KituraTests/TestResponse.swift

```diff
@@ -101,6 +103,38 @@ class TestResponse: KituraTest {
             })
         }
     }
+    
+    func testOptionalStringNilResponse() {
+        performServerTest(router) { expectation in
+            self.performRequest("post", path:"/sendNilString", callback: { response in
+                XCTAssertEqual(response?.statusCode, HTTPStatusCode.OK, "HTTP Status code was \(String(describing: response?.statusCode))")
+                
+                do {
+                    let body = try response?.readString()
+                    XCTAssertNil(body)
```

Comment:
I thought in the nil case we were sending the empty string - so wouldn't we expect to get an empty body back?